lib/streamDAO.js

```diff
@@ -309,21 +309,22 @@ module.exports = function(mongoClient){
       mongoClient.withCollection('summary', cb, function(coll, cb){
         let opts = { upsert : true };
         let timestamp = new Date();
-        let update = { "$set": {"dates.outdatedSince": timestamp} };
+        let update = { "$set": {"dates.outdatedSince": timestamp,
+                                          "dates.hasOutdatedSince": true} };
         let selector = { "userId": userId, "type": typ };
 
         coll.findOne(selector, function(err, summary){
           if (err != null) {
-            cb(err, typ);
```

Comment:
I see we're dropping the `type` from the callback signature, but it looks like the `log.warn` in databroker (https://github.com/tidepool-org/jellyfish/blob/47310ddaff988dca4c98254e2ff1b6003a9a5a48/lib/dataBroker.js#L117-L121) is still expecting it to be present.

Reply:
good catch, fixed.